js/forum/src/components/EditUserModal.js

```diff
@@ -100,6 +111,24 @@ export default class EditUserModal extends Modal {
     );
   }
 
+  activate() {
+    this.loading = true;
+    const data = {
+      username: this.username(),
+      isActivated: true,
+    };
+    this.props.user.save(data, {errorHandler: this.onerror.bind(this)})
+      .then(() => {
+        this.isActivated(true);
+        this.loading = false;
+        m.redraw();
+      })
```

Comment:
Indeed as dav-is's saying, it is a button, not a checkbox.

I initially designed it as a checkbox, but then realised it's almost nonsense to 'deactivate' the user.
As a button, it should take effects immediately.